src/main/java/net/craftcitizen/imagemaps/ImageMaps.java

```diff
@@ -370,7 +371,28 @@ private PlacementResult placeImage(Player player, Block block, BlockFace face, P
         
         return PlacementResult.SUCCESS;
     }
-    
+
+    public boolean deleteImage(String filename) {
+        File file = new File(getDataFolder(), IMAGES_DIR + File.separatorChar + filename);
+        boolean fileDeleted = false;
+        if (file.exists()) fileDeleted = file.delete();
+        imageCache.remove(filename.toLowerCase());
+        Iterator<Entry<ImageMap, Integer>> it = maps.entrySet().iterator();
+        while (it.hasNext()) {
+            Entry<ImageMap, Integer> entry = it.next();
+            ImageMap imageMap = entry.getKey();
+            if (!imageMap.getFilename().equalsIgnoreCase(filename)) continue;
+            Integer id = entry.getValue();
+            @SuppressWarnings("deprecation")
+            MapView map = Bukkit.getMap(id);
+            if (map == null) continue;
+            map.getRenderers().forEach(map::removeRenderer);
+            it.remove();
+        }
+        saveMaps();
+        return fileDeleted;
+    }
```

Comment:
Please avoid single line `if`, they make code very difficult to read.

Reply:
Added brackets and some spacing